### DISCUSSION TITLE
What are the dimensions of the distance tube?

### DISCUSSION THREAD

**Discussion by mrdago** [jomjol/AI-on-the-edge-device#1564] [General]
This is a really awesome project and would like to build this solution to integrate it in my SmartHome. Unfortunately I don't own a 3D-Printer. Therefore I'm thinking to trim a waste pipe into the right dimensions for use of the distance tube. Sadly I could not found any information about the size, the dimensions of that part.
I highly appreciate if someone could provide me the details of that component!
Thanks in advance!

**Comment by jomjol** [comment]
Go as close as possible, but with all values visible to the surface. The bigger the image, the better.  You can try by just holding it above the meter. Usually it is between 5 and 10 cm (you have to adjust the focus in any case).

**Reply by jurasjo** [reply]
Look at my pictures, maybe you can get some inspiration. I have used parts from an old hand water pump (a blue pipe and a cap), small protoboard and double sided tape. It works pretty well. The dimension of pipe is must be chosen according to your meter. Mine is ~80mm externally.
![314509934_1627551004313779_258133693612157671_n](https://user-images.githubusercontent.com/19259405/207218655-b3d288f1-e770-4e49-89ba-c0cd52b7780c.jpg)
![313924146_1823636281313717_1602341559975317389_n](https://user-images.githubusercontent.com/19259405/207218660-f0d230ba-7756-4a2e-9de5-2a8e3ed7c8a0.jpg)

**Comment by mrdago** [comment]
Ok, thanks a lot for your fast reply. I've ordered all required parts to start implementation between Chrismas and new Year.
Best regards!
